packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -108,14 +108,15 @@ class BlockListBlock extends Component {
 			getStylesFromColorScheme,
 			isSmallScreen,
 			isRootListInnerBlockHolder,
+			isUnregisteredBlock,
 		} = this.props;
 
 		const fullSolidBorderStyle = { // define style for full border
 			...styles.fullSolidBordered,
 			...getStylesFromColorScheme( styles.solidBorderColor, styles.solidBorderColorDark ),
 		};
 
-		if ( hasChildren ) { // if block has children apply style for selected parent
+		if ( hasChildren && ! isUnregisteredBlock ) { // if block has children apply style for selected parent
```

Comment:
> One approach that initially seems to be working is to change the hasChildren check instead:
> 
> const hasChildren = ! isUnregisteredBlock && !! getBlockCount( clientId );

@koke I did that check there, so need to find another way for styling unsupported block.